src/component/feedback/FeedbackCES/sendFeedbackInput.ts

```diff
@@ -0,0 +1,40 @@
+import { IFeedbackCESForm } from 'component/feedback/FeedbackCES/FeedbackCESForm';
+
+interface IFeedbackEndpointRequestBody {
+    source?: 'docs' | 'app' | 'app:segments';
```

Comment:
The app should never send `docs` feedback, should it? I suggest removing `docs` from the type definition in the app, so as to avoid possible misfires 💁🏼

Reply:
And in doing so, I'd also make `source` a required property.